catalog/lib/common/helper.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:catalog/extensions/color_extensions.dart';
+import 'package:flutter/material.dart';
+
+Color? _getMaterialStatesColor(
```

Comment:
Make this named parameters optional like this so we can change just the parameter we need, like this example: 
```
/// Sets the [bold] and [hidden] flags ...
void enableFlags({bool bold = false, bool hidden = false}) {...}

// bold will be true; hidden will be false.
enableFlags(bold: true);
```